{
  "Severity": "HIGH",
  "UpdatedAt": "2018-11-30T19:54:29Z",
  "Package": {
    "Ecosystem": "PIP",
    "Name": "python-gnupg"
  },
  "Advisory": {
    "DatabaseId": 950,
    "Id": "MDE2OlNlY3VyaXR5QWR2aXNvcnlHSFNBLWMyZngtOHI3Ni1naDM2",
    "GhsaId": "GHSA-c2fx-8r76-gh36",
    "References": [
      {
        "Url": "https://nvd.nist.gov/vuln/detail/CVE-2013-7323"
      },
      {
        "Url": "https://github.com/advisories/GHSA-c2fx-8r76-gh36"
      },
      {
        "Url": "http://seclists.org/oss-sec/2014/q1/243"
      },
      {
        "Url": "http://seclists.org/oss-sec/2014/q1/244"
      },
      {
        "Url": "http://seclists.org/oss-sec/2014/q1/294"
      },
      {
        "Url": "http://www.debian.org/security/2014/dsa-2946"
      },
      {
        "Url": "https://code.google.com/p/python-gnupg"
      },
      {
        "Url": "https://github.com/pypa/advisory-database/tree/main/vulns/python-gnupg/PYSEC-2014-89.yaml"
      }
    ],
    "Identifiers": [
      {
        "Type": "GHSA",
        "Value": "GHSA-c2fx-8r76-gh36"
      },
      {
        "Type": "CVE",
        "Value": "CVE-2013-7323"
      }
    ],
    "Description": "python-gnupg before 0.3.5 allows context-dependent attackers to execute arbitrary commands via shell metacharacters in unspecified vectors.",
    "Origin": "UNSPECIFIED",
    "PublishedAt": "2018-11-06T23:14:22Z",
    "Severity": "HIGH",
    "Summary": "High severity vulnerability that affects python-gnupg",
    "UpdatedAt": "2024-10-24T21:56:34Z",
    "WithdrawnAt": "",
    "CVSS": {
      "Score": 9.8,
      "VectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
    }
  },
  "Versions": [
    {
      "FirstPatchedVersion": {
        "Identifier": "0.3.7"
      },
      "VulnerableVersionRange": "\u003c 0.3.5"
    }
  ]
}